### PR TITLE
Add usecase to get the average number of participants in a visit

### PR DESCRIPTION
**Pull request by chubberlisk**
# What

Add usecase to get the average number of participants in a visit for a trust as well as small refactoring to use a helper for the event action names.

# Why

So we can display the metric on the trust admin dashboard.

# Screenshots

N/A

# Notes

Best reviewed commit by commit.

Would like feedback on:

- How do deal with the long contract test.
- Are there any edge cases I'm missing?

### CODE REVIEW

File: src/usecases/retrieveAverageParticipantsInVisit.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+    for (const [, sessionEvents] of Object.entries(eventsGroupedBySesssion)) {
+      const joinEvent = sessionEvents.find(
+        (event) => event.action === JOIN_VISIT
+      );
+      const leaveEvent = sessionEvents.find(
+        (event) => event.action === LEAVE_VISIT
+      );
+
+      if (joinEvent && leaveEvent) {
```
Comment (jivdhaliwal) [inline src/usecases/retrieveAverageParticipantsInVisit.js:43]: Do we need to check for both join and leave visit events? Have we considered simply counting the unique session IDs grouped by visit ID and then getting the average count from there ?

Comment (chubberlisk) [reply src/usecases/retrieveAverageParticipantsInVisit.js:43]: Nope, I didn't consider just using the session ID! I did consider grouping in the SQL statement like I said in the Notes but as a first pass I decided to write the simplest SQL statement first (as it's not something I'm very knowledgeable about so didn't want to get bogged down on perfecting a statement) and use JS to do the rest.

If doing things in Postgres is the way we want to go, could we 🍐 on it?

Comment (chubberlisk) [reply src/usecases/retrieveAverageParticipantsInVisit.js]: I think it turned out easier than I thought! Just pushed up https://github.com/madetech/nhs-virtual-visit/pull/455/commits/e03f0f20c13b09170b8b6490dbb8b06947915f1a, let me know if this is what you were thinking.

**Comment by chubberlisk** [comment]
Closing this in favour of https://github.com/madetech/nhs-virtual-visit/pull/456 which includes displaying the metric on the dashboard. Not much gain by splitting the feature into 2 PRs as it doesn't make it much bigger.
